components/script/dom/audiocontext.rs

```diff
@@ -48,12 +48,15 @@ pub struct AudioContext {
 impl AudioContext {
     #[allow(crown::unrooted_must_root)]
     // https://webaudio.github.io/web-audio-api/#AudioContext-constructors
-    fn new_inherited(options: &AudioContextOptions, pipeline_id: PipelineId) -> AudioContext {
+    fn new_inherited(
+        options: &AudioContextOptions,
+        pipeline_id: PipelineId,
+    ) -> Fallible<AudioContext> {
```

Comment:
@gterzian @jdm this is ready for review and depends on https://github.com/servo/media/pull/424